gocourse02/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	zookyper := Zookyper{
+		Name:   "Mukola",
+		IsBusy: false,
+	}
+	zookyper.PrintName()
+
+	cage := Cage{
+		Number:   1,
+		IsBroken: true,
+	}
+
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+	if !zookyper.IsBusy && cage.IsBroken {
+		cage.FixCage()
+	}
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+
+	var sirko = Animal{
+		Name:   "Sirko",
+		Number: 1,
+	}
+	fmt.Printf("Animal with name %s is istting in cage number %d\n", sirko.Name, sirko.Number)
+
+	var sirko_son = sirko.Duplicate("Sirko junior", 1)
```

Comment:
В Go-коді прийнятий `mixedCaps` для назв змінних, структур, констант:
```suggestion
	var sirkoSon = sirko.Duplicate("Sirko junior", 1)
```

Про це написано тут https://go.dev/doc/effective_go#mixed-caps
> Finally, the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.

Я це не розповідав на вебінарах, але пропоную відразу називати змінні правильно.